core/dbt/contracts/connection.py

```diff
@@ -178,6 +178,9 @@ def __post_serialize__(self, dct):
             )
         return dct
 
+    def acquire_handle(self, **kwargs):
+        raise NotImplementedError
```

Comment:
blocker: I don't think we need this anymore right?

Reply:
Ah, yeah, remnants from a previous implementation. I'll remove it.